src/ppx/scene/scene_material.cpp

```diff
@@ -108,6 +125,12 @@ scene::VertexAttributeFlags UnlitMaterial::GetRequiredVertexAttributes() const
     return attrFlags;
 }
 
+bool UnlitMaterial::HasTextures() const
+{
+    bool hasBaseColorTex = HasBaseColorTexture();
+    return hasBaseColorTex;
```

Comment:
nit: `return HasBaseColorTexture()` ?

Reply:
This is to make debugging a bit more convenient when needing to set a breakpoint to see the value. Otherwise it requires digging through the other function. I can remove it later, but would prefer it left in for now while the renderer is being written.